newm-server/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
remove this file

Reply:
Will remove it and add it to .gitignore file